app/models/api_model.php

```diff
@@ -61,7 +65,26 @@
                     $short_desc = $set_limit(80, $short_desc($translate));
 
                     if ($desc && $title && $short_desc) {
```

Comment:
tip: here we could just add the 3 values and check that the result is 3
```php
if ( ($desc +$title + $short_desc) < 3 {
    continue;
}
```
This way you would skip the loop early if the length constraints are not met and void an indentation level for all the subsequent code, making it more readable.